zookeeper-docs/src/main/resources/markdown/zookeeperMonitor.md

```diff
@@ -72,6 +73,125 @@ All the metrics are included in the `ServerMetrics.java`.
 
 - Now Prometheus will scrape zk metrics every 10 seconds.
 
+<a name="Alerting"></a>
```

Comment:
what about "Alerting with Prometheus" ?
because this is only about using Prometheus